app/assets/stylesheets/pageflow/themes/legacy/indicators.scss

```diff
@@ -0,0 +1,62 @@
+.entry {
+  .indicator {
+    color: #e6e6e6;
+
+    text-transform: uppercase;
+    text-shadow: 1px 1px 4px rgba(43, 43, 43, 1);
+    font-size: 0.92em;
+    letter-spacing: 0.2em;
+
+    padding-left: 40px;
+    font-family: sourcesanspro;
+    padding-top: 6px;
+
+    &:before {
+      background-size: auto auto;
+      background-position: top left;
+      background-repeat: no-repeat;
+      content: "";
+      width: 100%;
+      height: 100%;
+      left: 0;
+      top: 1px;
+      position: absolute;
+    }
+
+    @include phone {
+      text-shadow: 1px 1px 1px rgba(43, 43, 43, 1);
+      font-size: 0.8em;
+      letter-spacing: 0.1em;
+    }
+  }
+
+  .scroll_indicator {
+    &:before {
+      background-image: image-url("pageflow/themes/legacy/scroll_down_indicator.png");
+    }
+
+    &.invert {
+      text-shadow: 1px 1px 4px #FDFDFD;
```

Comment:
Color literals like `#FDFDFD` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FDFDFD` should be written as `#fdfdfd`